apps/indexer/src/lib/constants.ts

```diff
@@ -130,6 +130,7 @@ export const CONTRACT_ADDRESSES = {
   },
   [DaoIdEnum.OBOL]: {
     blockTime: 12,
+    tokenType: "ERC20",
```

Comment:
I believe this should be in the backend/indexer PR, as changes to files in `/scr/lib` will trigger re-indexing on PR merge

Reply:
Sure. But this branch will be merged there instead of on dev. So, there is no problem.